### PR TITLE
Add button to CONTAINER_TOOLBAR

### PR DESCRIPTION
**Pull request by Dark-Peace**
I added a button next to launch current scene and launch main scene, for easier access. It works just like pressing F9

### CODE REVIEW

File: addons/QuickSceneRunner/QuickSceneRunner.gd [hunk 1]
```diff
@@ -2,12 +2,20 @@
 extends EditorPlugin
 
 var dock: Control
+var button:Control
 
 func _enter_tree():
 	dock = preload("res://addons/QuickSceneRunner/QuickSceneRunner.tscn").instantiate()
 	dock.plugin = self
 	add_control_to_bottom_panel(dock, "Quick Scenes")
+	button = preload("res://addons/QuickSceneRunner/Button.gd").new()
```
Comment (KoBeWi) [inline addons/QuickSceneRunner/QuickSceneRunner.gd:11]:
```suggestion
	button = Button.new()
	button.pressed.connect(dock.run_scene)
```
The button script is not necessary at all.

File: addons/QuickSceneRunner/QuickSceneRunner.gd [hunk 2]
```diff
@@ -2,12 +2,20 @@
...
+	button.dock = dock
+	button.plugin = self
+	button.icon = get_editor_interface().get_base_control().theme.get_icon("TransitionSync", "EditorIcons")
```
Comment (KoBeWi) [inline addons/QuickSceneRunner/QuickSceneRunner.gd:14]:
```suggestion
	button.icon = get_editor_interface().get_base_control().get_theme_icon("TransitionSync", "EditorIcons")
```
This is how you get editor theme items.

File: addons/QuickSceneRunner/SceneRunner.gd [hunk 3]
```diff
@@ -53,7 +53,8 @@ func update_selected(scene: Control):
 	ProjectSettings.set_setting(SELECTED_SCENE_SETTING, scene.get_index())
 	ProjectSettings.save()
 
-func run_scene(scene: Control):
+func run_scene(scene: Control=null):
+	if scene == null: scene = shortcut_group.get_pressed_button().get_parent().get_parent().get_parent()
```
Comment (KoBeWi) [inline addons/QuickSceneRunner/SceneRunner.gd:57]: This will fail if there is no pressed button.
Also for readability:
```suggestion
	if scene == null:
		scene = shortcut_group.get_pressed_button().get_parent().get_parent().get_parent()

```

File: addons/QuickSceneRunner/SceneRunner.gd [hunk 4]
```diff
@@ -69,10 +70,10 @@ func edit_scene(scene: Control):
 
 func save_scenes():
 	var scene_list := PackedStringArray()
-
```
Comment (KoBeWi) [inline addons/QuickSceneRunner/SceneRunner.gd:72]: Unnecessary whitespace changes. I assume you have a setting enabled, but this shouldn't be commited.

**Comment by Dark-Peace** [comment]
Your changes were committed

**Comment by KoBeWi** [comment]
Sorry for the late reply, but you applied my suggestions without reading the comments. You also need to address the things I commented about.

Right now the plugin doesn't even work.

**Comment by Dark-Peace** [comment]
What comment ? "The button script is not necessary at all." ?
With the view of the code that doesn't include the next line that produces the very obvious error that could've just been avoided if it had been removed with the rest ?
That's fixed now. It works again.
You're welcome.

**Comment by KoBeWi** [comment]
I mean you should remove Button.gd, because it's unused.

**Comment by KoBeWi** [comment]
Looks good enough. Thanks.
